test/controllers/MessageControllerSpec.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import base.SpecBase
+import play.api.test.Helpers._
+import play.api.test.FakeRequest
+import org.mockito.Matchers.any
+import org.mockito.Mockito.{reset, when}
+import connector.DestinationConnector
+import org.scalatest.BeforeAndAfterEach
+import play.api.inject.bind
+import uk.gov.hmrc.http.{BadRequestException, HttpResponse, NotFoundException}
+
+import scala.concurrent.Future
+
+class MessageControllerSpec extends SpecBase with BeforeAndAfterEach {
+
+  private val mockConnector = mock[DestinationConnector]
+  private val application = applicationBuilder.overrides(bind[DestinationConnector].toInstance(mockConnector)).build()
+
+  override def beforeEach(): Unit = {
+    super.beforeEach()
+    reset(mockConnector)
+  }
+
+  "MessageController must" - {
+    "return ok for the message Type Goods release" in {
+      when(mockConnector.sendMessage(any(), any())(any(), any())).thenReturn(Future.successful(HttpResponse(OK)))
```

Comment:
Could we include the _X-Message-Type_ header/value to ensure it's being passed along and used in the sendMessage method?